src/components/pages/document-category.yml

```diff
@@ -0,0 +1,7 @@
+documentCategory:
+  documentCategoryNamePrefix:
+    en: Category
+    de: Kategorie
```

Comment:
I would suggest to keep consistent with the naming "Document category", even if it's longer 🤔

Reply:
Agreed, I changed it to "Document category" :+1: